clientgui/DlgEventLog.h

```diff
@@ -198,6 +202,9 @@ class CDlgEventLog : public DlgEventLogBase
 
     void                    ResetMessageFiltering();
 
+    void                    finderrormessages(bool isfiltered);
```

Comment:
Please rename these functions and their parameters to the 'camel-case' name convention:
`finderrormessages` -> `FindErrorMessages`
`findprojectmessages` -> `FindProjectMessages`
`isfiltered` -> `isFiltered`
This increases code readability.

Reply:
Noted on the whitespace (I'm not sure how I missed that, but I'll check).

As for the function names, I completely agree on the readability.  However, this statement is in the [Coding Style](https://github.com/BOINC/boinc/wiki/CodingStyle#namingl):

> Variable and function names are all lower case, with underscores to separate words.

I was just trying to follow this rule.  Let me know if I should proceed with camel-case or not.

Reply:
Please proceed with camel-case since all other functions are written this way. Thank you.

Reply:
Fixed camel case.